build/scripts/ImportTestDataInBcContainer.ps1

```diff
@@ -40,4 +40,4 @@ foreach ($app in (Get-BcContainerAppInfo -containerName $ContainerName -tenantSp
 }
 
 # Generate demo data in the container
-Invoke-ContosoDemoTool -ContainerName $parameters.ContainerName
\ No newline at end of file
+Invoke-ContosoDemoTool -ContainerName $parameters.ContainerName -SetupData $parameters.SetupData
```

Comment:
We don't seem to be using this switch internally - Should we be? 
https://dynamicssmb2.visualstudio.com/Dynamics%20SMB/_git/NAV?path=/Eng/Core/Scripts/BuildW1Database.ps1&version=GBmaster&_a=contents

We should try to match the way we run tests on BCApps with how we run them internally.